regulations/views/comment.py

```diff
@@ -23,6 +31,7 @@ def upload_proxy(request):
     url = s3.generate_presigned_url(
         ClientMethod='put_object',
         Params={
+            'ContentLength': size,
```

Comment:
If S3 errors when there's a content-length mismatch, we're golden. If it doesn't we might have a problem -- this file size data is coming from the UI (and therefore can't be trusted). Would you mind adding

```python
assert size > 0
```
just for good measure?